src/ResourceManager/Maps/Commands.Maps/MapsAccount/NewAzureMapsAccount.cs

```diff
@@ -52,7 +52,7 @@ public class NewAzureMapsAccount : MapsAccountBaseCmdlet
             Mandatory = true,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Maps Account Sku Name.")]
-        [ValidateSet("S0")]
+        [ValidateSet("S0", "S1")]
```

Comment:
We should move away from using ValidateSet to using PSArgumentCompleter - this allows users to tab through known vaues, but enter values outside the known values list.

Also, longer term, is this set of values somethign we should discover from API?

Reply:
@markcowl, made the change to use `PSArugementCompleter`